backend/app/services/adapters/task_kinds/helpers.py

```
@@ -0,0 +1,690 @@
+        updated_at = None
+        completed_at = None
+
+        if task_crd.status:
+            try:
+                if task_crd.status.createdAt:
+                    created_at = task_crd.status.createdAt
+                if task_crd.status.updatedAt:
+                    updated_at = task_crd.status.updatedAt
+                if task_crd.status.completedAt:
+                    completed_at = task_crd.status.completedAt
+            except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)